src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -330,7 +343,8 @@ export const waitForSaversUpdate = (txHash: string) =>
   poll({
     fn: () => getThorchainTransactionStatus(txHash),
     validate: status => Boolean(status && status === TxStatus.Confirmed),
-    interval: 30000,
+    interval: 60000,
+    // i.e max. 10mn from the first Tx confirmation
```

Comment:
def feels like we will want to make this dynamic by chainId (bitcoin blocks can take much longer even though on average they would make the cutoff)

Reply:
I'm not so sure about that one after testing this: from my testing, even "faster" chains (i.e Avalanche) still take a long time for things to be propagated in Midgard, so 60s still seem like a sane minimum for observed Tx, and we probably shouldn't rely on anything less than 10mn regardless of the chain

Reply:
Still going to explore this one for a follow-up, seems like we could definitely get away with way less than 60s for some chains - last I tested things propagated quite fast... As long as things are *deterministically* fast or slow (i.e depending on block times) we should be good to bring chain-specific programmaticness here